src/theme.js

```diff
@@ -0,0 +1,112 @@
+import theme from "@rebass/preset"
+
+const customTheme = {
+  ...theme,
+  breakpoints: [320, 768, 1024, 1440],
+  colors: {
+    darkBlue: "#0C1D60",
+    lightBlue: "#7BD5F5",
+    gold: "#F7D663",
+    white: "#FFF",
+  },
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "inherit",
+    monospace: "Menlo, monospace",
+  },
+  fontSizes: [
+    "0.75rem",
+    "1rem",
+    "1.333rem",
+    "1.777rem",
+    "2.369rem",
+    "3.157rem",
+    "4.209rem",
+    "5.61rem",
+    "7.478rem",
+    "9.969rem",
+    "13.288rem",
+    "17.713rem",
+    "23.612rem",
+    "31.475rem",
+    "41.956rem",
+    "55.927rem",
+    "74.551rem",
+    "99.376rem",
+  ],
+  fontWeights: {
+    lighter: 200,
+    light: 300,
+    normal: 400,
+    bold: 700,
+    bolder: 800,
+  },
+  lineHeights: {
+    body: 1.333,
+    heading: 1.33,
+  },
+  space: [
+    "0.75rem",
+    "1rem",
+    "1.333rem",
+    "1.777rem",
+    "2.369rem",
+    "3.157rem",
+    "4.209rem",
+    "5.61rem",
+    "7.478rem",
+    "9.969rem",
+    "13.288rem",
+    "17.713rem",
+    "23.612rem",
+    "31.475rem",
+    "41.956rem",
+    "55.927rem",
+    "74.551rem",
+    "99.376rem",
+  ],
+
+  // rebass variants
+  text: {
```

Comment:
Came back to the text styles I used in the design and I think these would be how they translate here:
```javascript
display: {
  fontFamily: "heading",
  fontWeight: "heading", // if you assign the heading fontWeight to be 900 above
  lineHeight: "heading",
  textTransform: "uppercase",
},
title: {
  fontFamily: "heading",
  fontWeight: "heading", // if you assign the heading fontWeight to be 900 above
},
subtitle: {
  fontFamily: "heading",
  fontWeight: "heading", // if you assign the heading fontWeight to be 900 above
  lineHeight: 1.777,
  textTransform: "uppercase",
  letterSpacing: "0.05em",
},
button: {
  fontFamily: "body",
  textTransform: "uppercase",
  letterSpacing: "0.1em",
}
```